app/src/main/java/com/kickstarter/ui/viewholders/HorizontalRewardViewHolder.kt

```diff
@@ -152,10 +160,15 @@ class HorizontalRewardViewHolder(private val view: View, val delegate: Delegate?
 
     private fun setMinimumText(@NonNull minimum: String) {
```

Comment:
This is now a misnomer, so it should be renamed to something like `setButtonText`.

Reply:
The `rewardMinimum` text is also being set here that's why I didn't change it but if it doesn't matter I'll rename it.

Reply:
There should be one method for setting the minimum text or just a single line in the subscribe block and a separate method for setting the button text.